graphqlhub-schemas/src/github.js

```diff
@@ -118,6 +131,26 @@ let CommitType = new GraphQLObjectType({
           return commit.commit && commit.commit.committer.date;
         }
       },
+      status : {
+        type : new GraphQLList(StatusType),
+        resolve(commit) {
+          const { username, reponame } = grabUsernameAndReponameFromURL(commit.url);
+          const { sha } = commit;
+
+          const statusForRepo = getStatusesForRepo(username, reponame, sha)
+              .then((statuses) => {
+                  return statuses.map(status => {
+                      const {
+                          id, url, state, context, description, target_url, updated_at
+                        } = status;
+                      return {
```

Comment:
Sure I just wanted to be careful :-) while I had some strange issues when using it with relay (#35)
I'll adapt it.